earth2mip/initial_conditions/gfs.py

```diff
@@ -180,6 +180,7 @@ def get_gfs_grib_file(
 def get(
     time: Union[datetime.datetime, None],
     gfs_channels: List[str],
+    download_gfs: bool,
```

Comment:
Personal preference would be to also name this `download_ic` for consistency.

Reply:
I considered that but thought it would be nice to have a lower-level flag, with the `download_ic` one that could be used for other DataSources like IFS (the hdf base class is handled differently with a `from_path` functionality that is not very well built out). Leaving as is for now, but we can certainly revisit later.